lib/facter/monit_version.rb

```diff
@@ -0,0 +1,14 @@
+# A facter fact to determine the installed version of monit.
+
+module Facter::Util::MonitVersion
+  class << self
+    def get_monit_version
+      monit_version = Facter::Util::Resolution.exec('monit -V 2>&1')
```

Comment:
@ghoneycutt Probably want to gate this behind a which check, so it doesn't run when monit's not installed:

```ruby
if Facter::Util::Resolution.which('monit')
  monit_version = Facter::Util::Resolution.exec('monit -V 2>&1')
  monit_version.match(/\d+\.\d+$/).to_s
end
```